configs/111m.yaml

```diff
@@ -16,26 +16,65 @@
 # the Cerebras-GPT 111M model except for the choice of dataset
 
 # model params
-width: 768
-depth: 10
-max_position_embeddings: 2048
-heads: 12
-
-# run params
-out_dir: model_dir
-# dataset can also point to either of the shakespeare datasets (see the data
-# directory for preprocessing info) or a custom dataset that you provide yourself
-dataset: /path/to/openwebtext # replace this with the correct absolute path before running
-batch_size: 120
-sequence_length: 2048
-num_steps: 9037
-checkpoint_steps: 1000
-learning_rate: 6.0e-4
-warmup_steps: 1525
-backend: "CSX"
-precision_opt_level: 1
-mount_dirs:
-  - /path/to/code/location
-  - /path/to/data/location
-python_paths:
-  - /path/to/code/location
+trainer:
```

Comment:
Thanks for this, I understand we were able to run on CPU with these changes, but we will need to also verify if we are able to run on CSX. @abhis-cerebras , can you please help here? These are changes made by @srinjoym-cerebras to port our giga gpt model to the new trainer flow. I think he was able to run it on CPU but we will need to verify these changes on CSX.